pkg/cmd/install/hubaddon/cmd.go

```diff
@@ -48,8 +48,9 @@ func NewCmd(clusteradmFlags *genericclioptionsclusteradm.ClusteradmFlags, stream
 	}
 
 	cmd.Flags().StringVar(&o.names, "names", "", "Names of the built-in add-on to install (comma separated). The built-in add-ons are: application-manager, governance-policy-framework")
-	cmd.Flags().StringVar(&o.values.Namespace, "namespace", "open-cluster-management", "Namespace of the built-in add-on to install. Default to open-cluster-management")
+	cmd.Flags().StringVar(&o.values.Namespace, "namespace", "open-cluster-management", "Namespace of the built-in add-on to install. Defaults to open-cluster-management")
 	cmd.Flags().StringVar(&o.outputFile, "output-file", "", "The generated resources will be copied in the specified file")
+	cmd.Flags().StringVar(&o.values.OcmVersion, "version", "v0.8.0", "The image version tag to use when deploying the hub add-on. Defaults to the latest released open-cluster-management version.")
```

Comment:
The text "Defaults to the latest released open-cluster-management version" indicates that it's the latest at the time of `clusteradm` execution but it's really at release time of `clusteradm`.

Reply:
> I think you could say it defaults to `v0.8.0` for now.

That's definitely the obvious and simplest answer, haha!